pydra/engine/helpers.py

```diff
@@ -57,9 +57,28 @@ def load_result(checksum, cache_locations):
     return None
 
 
-def save_result(result_path: Path, result):
-    with (result_path / "_result.pklz").open("wb") as fp:
-        cp.dump(result, fp)
+def save(task_path: Path, result=None, task=None):
+    """
+    Save ``Task`` object and/or results.
+
+    Parameters
+    ----------
+    task_path : Path
+        Write directory
+    result : Result
+        Result to pickle and write
+    task : Task
+        Task to pickle and write
+    """
+    if task is None and result is None:
+        raise ValueError("Nothing to be saved")
+    task_path.mkdir(parents=True, exist_ok=True)
```

Comment:
i would save tasks at a different location than the tasks output directory.

Reply:
i agree, i'd like to lump all slurm files (`task.pklz', `pyscript`, `batchscript`) in one location.